region/maridia/outer/Mt. Everest.json

```diff
@@ -762,6 +762,32 @@
       ],
       "flashSuitChecked": true
     },
+    {
+      "link": [1, 2],
+      "name": "R-Mode Spark Interrupt",
+      "entranceCondition": {
+        "comeInWithRMode": {}
+      },
+      "requires": [
+        "Gravity",
+        {"or": [
+          "h_CrystalFlashForReserveEnergy",
+          {"and": [
+            "h_RModeCanRefillReserves",
+            {"resourceMissingAtMost": [{"type": "PowerBomb", "count": 0}]},
+            {"partialRefill": {"type": "ReserveEnergy", "limit": 20}}
```

Comment:
with gravity + speed, you can kill 4 crab + 3 powamp, while keeping one crab alive